internal/webhook/module.go

```diff
@@ -129,8 +135,14 @@ func validateModuleLoaderContainerSpec(container kmmv1beta1.ModuleLoaderContaine
 			return fmt.Errorf("invalid regexp at index %d: %v", idx, err)
 		}
 
-		if container.ContainerImage == "" && km.ContainerImage == "" {
-			return fmt.Errorf("missing spec.moduleLoader.container.kernelMappings[%d].containerImage", idx)
+		if kmImg := km.ContainerImage; kmImg == "" {
+			if container.ContainerImage == "" {
+				return fmt.Errorf("missing spec.moduleLoader.container.kernelMappings[%d].containerImage", idx)
+			}
+		} else {
+			if !strings.Contains(kmImg, ":") && !strings.Contains(kmImg, "@") {
```

Comment:
WDYT about using NewTag and NewDigest from ithub.com/google/go-containerregistry? It provides a little bit more verifications on the input

Reply:
Fixed.